packages/calcite-components/src/components/checkbox/readme.md

```diff
@@ -1,5 +1,13 @@
 # calcite-checkbox
 
+The `calcite-checkbox` component is a small, square decision making components used for optional choice or multi-selectable lists.
+
+Checkboxes are designed to be used for "yes" and "no" interactions, or interactions where a choice is optional.
```

Comment:
This is weird, from the docs site I know, but maybe this can be rephrased. I'd remove most of this if possible.